tests/embind/imvu_test_adapter.js

```diff
@@ -563,10 +563,6 @@ function module(ignore, func) {
     assert['undefined'] = assert.equal.bind(null, undefined);
     assert.notUndefined = assert.notEqual.bind(null, undefined);
 
-    // ES3 synonyms
-    assert.false_ = assert['false'];
-    assert.true_ = assert['true'];
-
```

Comment:
I'm ok with not supporting ES3, but I don't understand what this code did. How is it helping ES3? Did `false` and `true` not exist then? But the code doesn't look like it's working around that.

Reply:
I suspect it is so that you can call `assert.false_` instead of `assert['false']` as `assert.false` is not allowed in ES3, but not sure.
Anyway, it doesn't seem to be used anywhere, so I thought it can be cleaned up.

Reply:
![image](https://user-images.githubusercontent.com/3396686/34068504-8cf09444-e27e-11e7-8294-03c7bd269bbd.png)

`true` and `false` were keywords in ES3 😄

Reply:
Makes sense, but is no code reading `assert.false_`? It seems like either this was never needed (even in ES3), or it's still needed.

Anyhow, probably not worth the time to figure this out - it's just in the bundled imvu test runner, which is used in a limited way, and those tests pass, so let's merge this.